_markbind/navigation/mainNav.md

```diff
@@ -23,56 +23,52 @@
   * [Rust]({{baseUrl}}/contents/rust/Rust.html)
   * [Swift]({{baseUrl}}/contents/swift/welcome-to-swift.html)
 
+* ****App Frameworks****
+  * [Android]({{baseUrl}}/contents/android/IntroToAndroid.html)
+  * [NodeJS]({{baseUrl}}/contents/node/nodejs.html)
+  * [VueJS]({{baseUrl}}/contents/javascript/Javascript-framework-VueJs.html)
+
 * ****SE Topics****
-  * Architecture
-    * [REST]({{baseUrl}}/contents/architecture/RESTArchitecturalStyle.html)
+  * [Agile Development]({{baseUrl}}/contents/projectManagement/AgileDevelopment.html)
   * Code Quality
     * [Code Quality Metrics]({{baseUrl}}/contents/codeQuality/CodeQualityMetrics.html)
+    * [Writing Testable Code]({{baseUrl}}/contents/codeQuality/writing-testable-code.html)
+  * [Databases]({{baseUrl}}/contents/databases/databases.html)
+  * [DevOps]({{baseUrl}}/contents/devops/DevOps.html)
   * [Documentation]({{baseUrl}}/contents/projectManagement/documentation.html)
   * [Functional Programming]({{baseUrl}}/contents/functionalProgramming/intro.html)
-  * Performance
-    * [Introduction to Performance Optimization]({{baseUrl}}/contents/performance/Performance.html)
-    * [Performance Profiling]({{baseUrl}}/contents/performance/PerformanceProfiling.html)
-  * Project Management
-    * [Agile Development]({{baseUrl}}/contents/projectManagement/AgileDevelopment.html)
-    * [DevOps]({{baseUrl}}/contents/devops/DevOps.html)
+  * [Regular Expressions]({{baseUrl}}/contents/regex/Regex.html)
   * Revision Control
     * [Best Practices with git]({{baseUrl}}/contents/revisionControl/bestPracticesGit.html)
-  * [Scalability]({{baseUrl}}/contents/scalability/scalable-development.html)
+  * Scalability and Performance
+    * [Introduction to Performance Optimization]({{baseUrl}}/contents/scalabilityAndPerformance/Performance.html)
+    * [Performance Profiling]({{baseUrl}}/contents/scalabilityAndPerformance/PerformanceProfiling.html)
+    * [Scalable Development]({{baseUrl}}/contents/scalabilityAndPerformance/scalable-development.html)
+  * Security
+    * [Cross Site Request Forgery (CSRF)]({{baseUrl}}/contents/security/crossSiteRequestForgery/crossSiteRequestForgery.html)
+    * [Cross Site Scripting (XSS)]({{baseUrl}}/contents/security/crossSiteScripting/crossSiteScripting.html)
+    * [Cryptography]({{baseUrl}}/contents/security/cryptography.html)
+    * [HTTPS]({{baseUrl}}/contents/security/Https.html)
+    * [SQL Injection]({{baseUrl}}/contents/security/sqlInjection.html)
+  * Software Architecture
+    * [REST]({{baseUrl}}/contents/softwareArchitecture/RESTArchitecturalStyle.html)
   * Static Analysis
```

Comment:
Should `Static Analysis` be under `Code Quality`?

Reply:
Yeah I think it makes sense - I previously considered it too but didn't feel super compelled at the time. Glad to know that one more person has similar thoughts! I'll go ahead with moving this 🙂